pkg/odo/cli/cli.go

```diff
@@ -89,6 +89,12 @@ func NewCmdOdo(name, fullName string) *cobra.Command {
 
 	rootCmd.PersistentFlags().Bool(genericclioptions.SkipConnectionCheckFlagName, false, "Skip cluster check")
 
+	// Add the machine readable output flag to all commands
+	// We use "flag" in order to make this accessible throughtout ALL of odo, rather than the
+	// above traditional "persistentflags" usage that does not make it a pointer within the 'pflag'
+	// package
+	flag.CommandLine.String("o", "", "Specify output format, supported format: json")
```

Comment:
@kadel With the way that global parameters are added with the standard flag library as well as the limitation on adding multiple flag aliases, either only `-o` OR `--output` can be added.

Would you be fine @kadel @dgolovin with removing `--output` (I dont see it being used much) with *just* `-o` ?